scripts/test-integrations

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
```

Comment:
Drive-by review and really nothing big. The shebang that should be used for bash is `#!/usr/bin/env bash`.

Reply:
Oh I'm very aware of that! But portability isn't really my concern here to be honest. And if tooling like shellcheck doesn't complain about this, then it's in practice not really something I'll persistently fix. Now if someone tries running these scripts and they fail for this reason, I'm happy to accept a patch.

Reply:
Makes sense!